### PR TITLE
Categoryid 

### PR DESCRIPTION
**Pull request by JaqKent**
implementation of the category of each product and the fix of the modal bug

### CODE REVIEW

File: src/components/CustomModal/styles.module.scss [hunk 1]
```diff
@@ -5,9 +5,10 @@
 .container {
   @include flex(center, center, column);
   background-color: rgba(black, 0.5);
-  min-width: 375px;
-  height: 100vh;
-  z-index: 9000;
+  min-width: 100%;
```
Comment (nathan-hyan) [inline src/components/CustomModal/styles.module.scss:8]: Sort css alphabetically

File: src/App/App.tsx [hunk 2]
```diff
@@ -9,12 +9,12 @@ import ProductIndex from "screens/productIndex";
 
 import { notifications } from "./constant";
 
-import ProductPage from "components/ProductPage";
+import ProductScreen from "components/ProductScreen";
+
 import SearchBar from "components/SearchBar";
 import Home from "screens/Home";
 import Footer from "components/Footer/index";
```
Comment (nathan-hyan) [inline src/App/App.tsx:16]: Sort imports

File: src/components/ProductScreen/index.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { useContext, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import LoadingSpinner from "components/LoadingSpinner";
+import ProductPage from "components/ProductPage";
+import { ProductsContext } from "context/Product";
+
+function ProductScreen() {
+  const { id } = useParams<any>();
```
Comment (nathan-hyan) [inline src/components/ProductScreen/index.tsx:8]: Replace any for string

File: src/components/productCard/styles.module.scss [hunk 4]
```diff
@@ -11,6 +11,15 @@
     width: 150px;
     height: 205px;
   }
+  @include responsive(md) {
+    border: 1px solid $black;
+    width: 300px;
```
Comment (nathan-hyan) [inline src/components/productCard/styles.module.scss:16]: Sort css

File: src/components/productCategory/styles.module.scss [hunk 5]
```diff
@@ -8,7 +8,7 @@
   .bar {
     @include flex(flex-start, center, row);
     background-color: $primary;
-    min-width: 375px;
+    min-width: 100%;
     height: 60px;
     gap: 10px;
     color: $white;
```
Comment (nathan-hyan) [inline src/components/productCategory/styles.module.scss:14]: Sort css

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Outstanding job

File: src/components/CustomModal/styles.module.scss (hunk 1 above)
Comment (JaqKent) [reply src/components/CustomModal/styles.module.scss:8]: done

File: src/App/App.tsx (hunk 2 above)
Comment (JaqKent) [reply src/App/App.tsx:16]: done

File: src/components/ProductScreen/index.tsx (hunk 3 above)
Comment (JaqKent) [reply src/components/ProductScreen/index.tsx:8]: done

File: src/components/productCard/styles.module.scss (hunk 4 above)
Comment (JaqKent) [reply src/components/productCard/styles.module.scss:16]: done

File: src/components/productCategory/styles.module.scss (hunk 5 above)
Comment (JaqKent) [reply src/components/productCategory/styles.module.scss:14]: done

**Pull request merged**